evaluator/pom.xml

```diff
@@ -58,28 +58,15 @@
             <artifactId>quick</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>info.cqframework</groupId>
-            <artifactId>fhir-r4</artifactId>
-            <scope>test</scope>
-        </dependency>
         <dependency>
             <groupId>info.cqframework</groupId>
             <artifactId>elm-fhir</artifactId>
             <scope>test</scope>
         </dependency>
-
-        <dependency>
-            <groupId>org.opencds.cqf.cql</groupId>
-            <artifactId>evaluator.jaxb-deps</artifactId>
-            <version>2.1.0-SNAPSHOT</version>
-            <type>pom</type>
-            <scope>test</scope>
-        </dependency>
         <dependency>
```

Comment:
Needed to remove this, cause getting the following error:
![image](https://user-images.githubusercontent.com/31050924/192666234-8ef01c26-7a39-442e-91cc-ad29d6996376.png)